### ISSUE TITLE
still very broken

### ISSUE THREAD

**Issue by alkamine** [K1rakishou/Kuroba-Experimental#1060]
Returning to the catalog by hitting the back navigation button still frezzes the whole app, requiring a restart. basic functionality to make the app usable is lacking.

voice to text keyboards still do not work

highliting text is near impossible.

**Comment by K1rakishou** [comment]
Yes, I'm aware. There is no need to create new issues. I'm working on a big update right now. These fixes will be a part of it.
